src/status_im2/contexts/wallet/create_account/backup_recovery_phrase/view.cljs

```diff
@@ -0,0 +1,93 @@
+(ns status-im2.contexts.wallet.create-account.backup-recovery-phrase.view
+  (:require
+    [quo.core :as quo]
+    [react-native.blur :as blur]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.contexts.wallet.create-account.backup-recovery-phrase.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+;; Temporary words
```

Comment:
can we put this in the temp file as it makes it easier to know what we have to clean later? 🙏

Reply:
Done